htdocs/PHP/internal_libraries/languages/page_content_fr.php

```diff
@@ -51,8 +51,8 @@
 $translation["landing_page_section_4_title"] = "Sûr";
 $translation["landing_page_section_4_description"] = "Votre note est chifrée en utilisant le <a href='https://wikipedia.org/wiki/Advanced_Encryption_Standard' target='_blank'>l'algorithme AES-256-CTR</a> et son code d'accès est généré de manière aléatoire. Et même si WeExpire est piraté, il est impossible d'accéder à vos notes car une fois générées, elles ne sont pas stockées sur WeExpire.";
 
-$translation["landing_page_section_5_title"] = "À but non lucratif";
-$translation["landing_page_section_5_description"] = "WeExpire est un projet à but non lucratif et est entièrement gratuit. Si vous avez envie d'aider le projet, vous pouvez tout de même contribuer avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.";
+$translation["landing_page_section_5_title"] = "Rapide et à but non lucratif";
+$translation["landing_page_section_5_description"] = "Sur WeExpire vous pouvez créer votre note d'urgence en moins de 3 minutes, gratuitement. Nous croyons que l'écriture d'un dernier message numérique doit être quelque chose de facile et abordable pour n'importe qui dans le monde. Si vous avez envie d'aider le projet, vous pouvez tout de même contribuer avec un <a href='https://www.buymeacoffee.com/ciccionamente' target='_blank'>don</a>.";
```

Comment:
Hey @eze-kiel, does this sound right in French?

Reply:
this one is perfect :ok_hand: